### DISCUSSION TITLE
In my PC .  i have a folder where i have local domain and inside i have Forest with files(ui.xml)  and .git file and Inside that  HEAD file) Whatever changes i do in my project ui is reflected in ui.xml  and when i commit it is updating HEAD . All this is happening on my Local PC only . Now my Requirement is to create A git diff of ui.xml and HEAD file using some Javascript code or Jquery code and Get result in form of Json or Jquery /Javascript format. I will use this diff file in my code  Can someone help me on this

### DISCUSSION THREAD

**Discussion by ghost** [community/community#52033] [Programming Help]
### Select Topic Area

Question

### Body

I have in my PC with  local Domain created. Now i have a folder where i have local domain and inside i have Forest with files(ui.xml)  and .git 9Inside HEAD file)
Whatever changes i do is reflected in ui.xml  and when i commit it is updating HEAD .
Now my Requirement is to create A git diff of ui.xml and HEAD file using some Javascript code or Jquery code and Get result in form of Json or Jquery /Jawascript format.
Can someone help me on this

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @PrabodhMohan ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
